### PR TITLE
Changed Meter to accept a single value property

### PR DESCRIPTION
**Pull request by ericsoderberghp**
#### What does this PR do?

Changed Meter to accept a single value property and a color property.
This makes it easier to use for the 80% case where there is only one value.

#### Where should the reviewer start?

index.d.ts

#### What testing has been done on this PR?

Revised the "Bar" storybook to use it.
Added a unit test.

#### How should this be manually tested?

Bar story

#### Do the grommet docs need to be updated?

automatic

#### Should this PR be mentioned in the release notes?

yes

#### Is this change backwards compatible or is it a breaking change?

backwards compatible

### CODE REVIEW

**Review by ShimiSun — COMMENTED** [review]
Why not leverage the existing values prop to accept a single value? 
I’m concerned similar name props will cause confusion of when to use which prop.
same for leveraging `background` vs creating a new `color` prop.

**Comment by ericsoderberghp** [comment]
> same for leveraging `background` vs creating a new `color` prop.

`background` is the "track", `color` is the value piece only. So, a value of 0 will show no color and a value of 100 will show no background.

**Comment by ericsoderberghp** [comment]
> Why not leverage the existing values prop to accept a single value?

Something just feels weird about `values={20}`. Trying to figure out if there's another way to address it. We're a bit stuck in that the current API is more oriented towards the 20% use cases than the 80% ones. :(

**Comment by ShimiSun** [comment]
One thought, deprecate `values` and use `value` instead. So any calls to `values` will fall back to `value` for backward compatibility.

File: src/js/components/Meter/index.d.ts [hunk 1]
```diff
@@ -11,13 +11,15 @@ export interface MeterProps {
...
   background?: string | { color?: string; opacity?: OpacityType };
+  color?: string;
   gridArea?: GridAreaType;
   margin?: MarginType;
   max?: number;
   round?: boolean;
   size?: 'xsmall' | 'small' | 'medium' | 'large' | 'xlarge' | 'full' | string;
   thickness?: 'xsmall' | 'small' | 'medium' | 'large' | 'xlarge' | string;
   type?: 'bar' | 'circle';
+  value?: number;
```
Comment (taysea) [inline src/js/components/Meter/index.d.ts:22]: If I have a single value but want to have so `onHover` or `onClick` behavior, do I still have to use `values`? Or does this fall outside of the 80% use case where people just want to display a single value and no interaction is desired

Comment (ericsoderberghp) [reply src/js/components/Meter/index.d.ts:22]: For that scenario, the default react/browser behavior of event listeners on the DOM element will work without grommet needing to do anything. Although, for a single value Meter, onClick might be better represented in the DOM via `<Button onClick={}><Meter /></Button>`.

**Comment by ericsoderberghp** [comment]
> One thought, deprecate `values` and use `value` instead. So any calls to `values` will fall back to `value` for backward compatibility.

After thinking about this some more, I'm inclined to keep them separate, as I think this aligns more closely with how we think. It also aligns with how the `date` and `dates` properties work in Calendar.

**Comment by ShimiSun** [comment]
> > One thought, deprecate `values` and use `value` instead. So any calls to `values` will fall back to `value` for backward compatibility.
> 
> After thinking about this some more, I'm inclined to keep them separate, as I think this aligns more closely with how we think. It also aligns with how the `date` and `dates` properties work in Calendar.

Interesting comparison, I had an in mind similar comparison of DateInput `value` which could be a single date string or array of string date, similar to how I thought Meter `value` should work. 
Do you think Calendar `date` and `dates` is a better comparison than DateInput `value`?

**Comment by ericsoderberghp** [comment]
> > > One thought, deprecate `values` and use `value` instead. So any calls to `values` will fall back to `value` for backward compatibility.
> > 
> > 
> > After thinking about this some more, I'm inclined to keep them separate, as I think this aligns more closely with how we think. It also aligns with how the `date` and `dates` properties work in Calendar.
> 
> Interesting comparison, I had an in mind similar comparison of DateInput `value` which could be a single date string or array of string date, similar to how I thought Meter `value` should work.
> Do you think Calendar `date` and `dates` is a better comparison than DateInput `value`?

Well, DateInput needs to be tightly aligned with how form input controls work, with `value` and `onChange`, as with TextInput, Select, CheckBoxGroup, etc.

Meter feels more akin to the other Visualization components, to me.

**Pull request merged**